docs/THIRDPARTY

```diff
@@ -253,6 +253,11 @@ Copyright: Google, Inc.
 License: Apache-2.0
 Comment: Material volume up icon (rounded)
 
+Files: web/shared/icons/recent.svg
+Source: https://lucide.dev/icons/history
+Copyright: 2013-2022 Cole Bemis
+License: ISC License
```

Comment:
I seem to recall that we list ISC License as the compatible Expat, but Tim will be able to chime in on that.